src/app_service/service/stickers/service.nim

```diff
@@ -403,20 +398,23 @@ QtObject:
       slot: "onStickerPackInstalled",
       chainId: self.networkService.getNetworkForStickers().chainId,
       packId: packId,
-      hasKey: self.marketStickerPacks.hasKey(packId)
     )
     self.threadpool.start(arg)
 
   proc onStickerPackInstalled*(self: Service, installedPackJson: string) {.slot.} =
     let installedPack = Json.decode(installedPackJson, tuple[packId: string, installed: bool])
     if installedPack.installed:
+      if self.marketStickerPacks.hasKey(installedPack.packId):
+        self.marketStickerPacks[installedPack.packId].status = StickerPackStatus.Installed
       self.events.emit(SIGNAL_STICKER_PACK_INSTALLED, StickerPackInstalledArgs(
         packId: installedPack.packId
       ))
+    else:
+      error "Sticker pack did not get installed", packId = installedPack.packId
```

Comment:
I'd appreciate quick return with inverted condition, but this is a minor thing ofc 🙂 
```nim
    if not installedPack.installed:
      error "Sticker pack did not get installed", packId = installedPack.packId
      return
```

Reply:
Good point. I agree, but also, I'm lazy and the stickers service is pretty messed up, so eff it, I'll merge